src/main/java/io/mosip/mimoto/util/Utilities.java

```diff
@@ -182,8 +182,11 @@ public String getIssuersConfigJsonValue() {
     public String getTrustedVerifiersJsonValue() {
         return getJson(trustedVerifiersJsonString, trustedVerifiersPath);
     }
-    public String getCredentialSupportedTemplateString() {
-        return getJson(credentialTemplateHtmlString, credentialTemplatePath);
```

Comment:
This change is not for locale story ideally. I

Reply:
Yeah, this change of adding issuerId & crednetialType is to support multiple templates in future, where name of template would include issuerId & credentialType.